Magento/Tests/Templates/ThisInTemplateUnitTest.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+echo $block->escapeHtml($block->getGroupCode());
+echo $this->escapeHtml($this->getGroupCode());
+$this->foo();
+$this->helper();
```

Comment:
I believe this is not allowed either. Please provide source of information which states that this is allowed.

Reply:
Fixed sniff.